docs/releases/2.7.1.rst

```diff
@@ -105,8 +124,62 @@ Command changes and additions
 - Added a :option:`--no-rq` option to run commands in a single process without
   using RQ workers.
 
-...and lots of refactoring, cleanups to remove old Django versions specifics,
-improved documentation and of course, loads of bugs were fixed.
+- Now it is possible specify the parameters to set up your database directly
+  through :djadmin:`init command <init>`.
+
+
+Editor
+------
+
+- Editor now request confirmation before navigating away from modified units in
+  order to prevent data loss. This also includes non-saved comments. Going to
+  the previous, next, and a specific unit will trigger the prompt, as well as
+  changing filters or searching. It is also triggered by typing a different
+  URL, reloading the page or closing the browser window.
+
+- Fixed issue that didn't allow users with only just suggestion rights to send
+  suggestions.
+
+- Suggestion related events are now displayed on the timeline.
+
+- Critical and not critical failing checks are now displayed separately in the
+  editor.
+
+- Potential errors when managing the suggestions are now displayed to users.
+
+- Now users can reject their own suggestions even if they don't have enough
```

Comment:
I'd say this is a fixed regression where users couldn't reject their own suggestions without reviewing rights.